packages/oc-template-es6-compiler/lib/viewTemplate.js

```diff
@@ -1,7 +1,7 @@
 const viewTemplate = ({ css, bundle }) => `function(model){
-  var template = ${bundle};
+  ${bundle};
```

Comment:
I don't get this change

Reply:
webpack 5 changes this, and automatically creates a "module" variable for you that has this.